test/aeso_compiler_tests.erl

```diff
@@ -379,7 +379,7 @@ failing_contracts() ->
         [<<?Pos(2, 11)
            "Include of 'included.aes' at line 2, column 11\nnot allowed, include only allowed at top level.">>,
          <<?Pos(3, 13)
-           "Nested namespace not allowed\nNamespace 'Foo' at line 3, column 13 not defined at top level.">>])
+           "Nested namespaces are not allowed\nNamespace 'Foo' at line 3, column 13 not defined at top level.">>])
     , ?TYPE_ERROR(bad_address_literals,
```

Comment:
There were quite a few new error messages - a test case for each would be nice :-)

Reply:
Test cases are missing not only for the new messages. Imo this is a separate job to do.

Reply:
Two wrongs doesn't equal one right, or whatever the saying is :-)

Reply:
If you say so. I will extend the scope of the PR and add tests for all the error messages

Reply:
Done